lib/hanami/commands/console.rb

```diff
@@ -60,7 +60,7 @@ def engine
       # @since 0.1.0
       # @api private
       def engine_lookup
-        (ENGINES.find { |_, klass| Object.const_defined?(klass) } || DEFAULT_ENGINE).first
+        (ENGINES.find { |_, klass| Object.const_defined?(klass) } || [DEFAULT_ENGINE]).first
```

Comment:
What do you think, may be is better to set `DEFAULT_ENGINE` constant to array of engine? Like this:
``` ruby
DEFAULT_ENGINE = ['irb'].freeze
```